### PR TITLE
changes to user research links

### PR DESCRIPTION
**Pull request by skipster2k2**

### CODE REVIEW

File: digital_register_view/digital_register_main.html [hunk 1]
```diff
@@ -53,7 +53,7 @@ <h2 class="page-header">What has the Digital Register View Service been built on
   <p>DRV has been developed on new platforms and technology stacks for Land Registry. We have continued to develop primarily in the python programming language (and ruby for automated testing). <a href="https://github.com/LandRegistry">Our code is primarily in the open</a> (apart from where the code contains access to official-sensitive data or commercial intellectual property).
   </p>
   <h4>High level DRV Architecture</h4>
-  <a href="/images/Rough_Architecture.jpg">
-    <img src=/images/Rough_Architecture.jpg alt="High level architecture diagram" class="img-thumbnail" height="1334" width="750"/>
+  <a href="/images/DRV_Application_Architecture.jpg">
+    <img src="/images/DRV_Application_Architecture.jpg" alt="High level architecture diagram" class="img-thumbnail" />
```
Comment (woodingd) [inline digital_register_view/digital_register_main.html]: Old image had height and width attributes. Just a sanity check on whether the new image needs them or not.

**Comment by woodingd** [comment]
check comment, otherwise lgtm :+1:

File: digital_register_view/digital_register_main.html (hunk 1 above)
Comment (andymantell) [inline digital_register_view/digital_register_main.html]: The main reason for height/width is to prop the page open while the image is loading, that's not an issue here on a non user facing site. Carry on I reckon

**Comment by andymantell** [comment]
:+1:

**Pull request merged**